### PR TITLE
[IGVM] add config file and ec private key as input arguments to igvm script

### PR DESCRIPTION
**Pull request by manojrupireddy**
Developer can specify isolated guest VM signing time configurations as an an input json file.
For signing SNP_ID_BLOCK, developer need to supply ec key as input to igvmfile.py script.

### CODE REVIEW

File: scripts/igvmfile.py [hunk 1]
```diff
@@ -1588,15 +1699,28 @@ def load_kernel(kernel, cmdline, ramdisk, vtl):
     parser.add_argument('-append', type = str, metavar = 'cmdline')
     parser.add_argument('-rdinit', type = argparse.FileType('rb'), metavar = 'ramdisk')
     parser.add_argument('-vtl', type = int, metavar = '2', help = 'highest vtl', required = True)
+    parser.add_argument('-config_file', type = str, help = 'igvm config file', required = False )
+    parser.add_argument('-sign_key', type = argparse.FileType('rb'), help = 'private signing key', required = True)
```
Comment (wdcui) [inline scripts/igvmfile.py]: If no sign_key is given, can you have the script to generate a key as before?

File: scripts/igvmfile.py [hunk 2]
```diff
@@ -1588,15 +1699,28 @@ def load_kernel(kernel, cmdline, ramdisk, vtl):
     parser.add_argument('-append', type = str, metavar = 'cmdline')
     parser.add_argument('-rdinit', type = argparse.FileType('rb'), metavar = 'ramdisk')
     parser.add_argument('-vtl', type = int, metavar = '2', help = 'highest vtl', required = True)
+    parser.add_argument('-config_file', type = str, help = 'igvm config file', required = False )
+    parser.add_argument('-sign_key', type = argparse.FileType('rb'), help = 'private signing key', required = True)
+    #sample key can be generated using `openssl ecparam -out ca.key -name secp384r1 -genkey`
     args = parser.parse_args()
+    
+    if args.config_file:
```
Comment (wdcui) [inline scripts/igvmfile.py:1706]: Would you like to check in a config file or the default one is good?

File: scripts/igvmfile.py (hunk 1 above)
Comment (manojrupireddy) [reply scripts/igvmfile.py]: Yes, I will do that

File: scripts/igvmfile.py (hunk 2 above)
Comment (manojrupireddy) [reply scripts/igvmfile.py:1706]: I did not understand this comment/question. Can you explain. @wdcui

Comment (wdcui) [reply scripts/igvmfile.py:1706]: Say Alvin wants to create an IGVM file with a TEI config, when would he get the config file?  If such a config file is needed, maybe you can check it into this repo for now so it's self-contained?

Comment (manojrupireddy) [reply scripts/igvmfile.py:1706]: Got it. I added sample file in the form of comments at line number 133. But yes, I will check-in a sample config file. Also, specifying config file is an optional one. If no config file is provided, default values will be taken in the script.

**Pull request merged**